src/navigation/MainTabNavigator.tsx

```diff
@@ -114,6 +130,14 @@ const MainTabNavigator: FunctionComponent = () => {
           tabBarIcon: ExposureHistoryIcon,
         }}
       />
+      <Tab.Screen
```

Comment:
We will need to be able to conditionally render this based off of a config var as not every HA will want/need this. We also want to be able to cut release builds potentially while this is under active development.  This could come in a followup pr though.

Reply:
Sounds good will handle in this PR.